README.md

```diff
@@ -53,15 +32,15 @@ You can simply start the frontend:
 npm run serve
 ```
 
-ATTENTION: Lunie requires a backend. This is currently not yet public.
+The front end will run on its own, however if you want to feed it data to interact with, please read on below.
 
 ### Run local testnet
 
-#### Docker
+ATTENTION: Lunie requires a backend running to operate. We do not offer a publicly accessible backend, however you can follow the instructions below to run a local testnet backend.
 
-To run a local testnet for Lunie you will need [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) installed. You can find instructions to install Docker [here](https://www.docker.com/get-docker). You can find instructions to install Docker-Compose [here](https://docs.docker.com/compose/install/).
+[comment]: <> (removed docker section as it's not needed to complete installion and running Lunie)
```

Comment:
we can delete this line i think @jrmoreau

Reply:
OK, got it. I was looking up for markdown commenting styles that wouldn't show on the front end. Wanted to comment for context but I can remove it.

Reply:
cool. i didn't know there was such a thing! yeah, can be removed though!